test/DocumentFormat.OpenXml.Framework.Tests/OpenXmlPartTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿using DocumentFormat.OpenXml.Features;
+using DocumentFormat.OpenXml.Packaging;
+using NSubstitute;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace DocumentFormat.OpenXml.Framework.Tests
+{
+    public class OpenXmlPartTests
+    {
+
+        [Fact]
+        public void ExtensionTest()
```

Comment:
Can you devise some permutations of things here to validate expected behavior? i.e. what if there is no extension defined or if IPartExtensionFeature has an extension defined on it?

Reply:
I added tests for no extension defined,  extension defined on IPartExtensionFeature, and IsContentTypeFixed == true